lib/ansible/cli/galaxy.py

```diff
@@ -1006,20 +1007,25 @@ def execute_init(self):
             description='your {0} description'.format(galaxy_type),
             ansible_plugin_list_dir=get_versioned_doclink('plugins/plugins.html'),
         )
-        if galaxy_type == 'role':
-            inject_data.update(dict(
-                author='your name',
-                company='your company (optional)',
-                license='license (GPL-2.0-or-later, MIT, etc)',
-                role_name=obj_name,
-                role_type=context.CLIARGS['role_type'],
-                issue_tracker_url='http://example.com/issue/tracker',
-                repository_url='http://example.com/repository',
-                documentation_url='http://docs.example.com',
-                homepage_url='http://example.com',
-                min_ansible_version=ansible_version[:3],  # x.y
-                dependencies=[],
-            ))
+
+        if galaxy_type == "role":
+            semantic_version = SemanticVersion.from_loose_version(LooseVersion(ansible_version))
+            min_ansible_version = f"{semantic_version.major}.{semantic_version.minor}"
```

Comment:
In triage we discussed that perhaps this should include the patch as well. It looks like historically min_ansible_version was hardcoded in ansible-galaxy to 1.2 and 2.4, before taking the first 3 characters (which worked through 2.9).  I think Galaxy supports a specific patch (as long as the version is quoted) so it can be edited after `ansible-galaxy role init`.  I'm in favor of this as-is since it's backwards compatible and seems like a good general default, but it could use another review.